explanation/performance.rst

```diff
@@ -38,10 +38,12 @@ following sections.
 * Obtain the hierarchical map of key computing elements using :ref:`hwloc and lstopo <perf-tune-hwloc>`
 * Check and control CPU governors, power and frequency with :ref:`cpupower <perf-tune-cpupower>`
 * Dynamic, adaptive system tuning :ref:`with TuneD <perf-tune-tuned>`
+* Profile-Guided Optimization :ref:`PGO <perf-pgo>`
```

Comment:
I actually think this new page should be at the top of the list

Reply:
ACK.  Moved to the top :-).